src/test/java/api/requests/RequestsAPICreationTests.java

```diff
@@ -2982,6 +2984,48 @@ void itemCheckOutShouldNotAffectRequestAssociatedWithAnotherItemOfInstance() {
       is(RequestStatus.OPEN_NOT_YET_FILLED.getValue()));
   }
 
+  @Test
+  public void itemCheckOutRecallRequestCreatedShouldProduceNotices() {
+    UUID recallToLoaneeTemplateId = UUID.randomUUID();
+    JsonObject recallToLoaneeConfiguration = new NoticeConfigurationBuilder()
+      .withTemplateId(recallToLoaneeTemplateId)
+      .withEventType(NoticeEventType.ITEM_RECALLED.getRepresentation())
+      .create();
+
+    NoticePolicyBuilder noticePolicy = new NoticePolicyBuilder()
+      .withName("Policy with recall notice")
+      .withLoanNotices(singletonList(recallToLoaneeConfiguration));
+
+    useFallbackPolicies(
+      loanPoliciesFixture.canCirculateRolling().getId(),
+      requestPoliciesFixture.allowAllRequestPolicy().getId(),
+      noticePoliciesFixture.create(noticePolicy).getId(),
+      overdueFinePoliciesFixture.facultyStandard().getId(),
+      lostItemFeePoliciesFixture.facultyStandard().getId());
+
+    configurationsFixture.enableTlrFeature();
+    ItemResource smallAngryPlanet = itemsFixture.basedUponSmallAngryPlanet();
+    checkOutFixture.checkOutByBarcode(smallAngryPlanet, usersFixture.jessica());
+
+    IndividualResource requester = usersFixture.steve();
+    ZonedDateTime requestDate = ZonedDateTime.of(2017, 7, 22, 10, 22, 54, 0, UTC);
+
+    final IndividualResource request =
```

Comment:
The variable is not used later, could we delete it please?
Also please, recheck RequestBuilder, as we need to indicate request level etc.

Reply:
done, thanks